packages/app/src/specs/banners/CreateOrganizationBanner.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <TrackedBanner
+    :banner-id="BannerIds.ACI_082022_CREATE_ORG"
+    :model-value="modelValue"
+    data-cy="create-organization-banner"
+    status="info"
+    :title="t('specPage.banners.createOrganization.title')"
+    class="mb-16px"
+    :icon="OrganizationIcon"
+    dismissible
+    @update:model-value="value => emit('update:modelValue', value)"
+  >
+    <p class="mb-24px">
+      {{ t('specPage.banners.createOrganization.content') }}
+    </p>
+
+    <ExternalLink
+      class="border rounded mx-auto outline-none bg-indigo-500 border-indigo-500 text-white max-h-60px py-6px px-16px inline-block hocus-default hocus:no-underline"
+      :href="createOrganizationUrl"
+      :include-graphql-port="true"
+      data-cy="create-organization-button"
+    >
+      <i-cy-office-building_x16 class="inline-block icon-dark-white" />
```

Comment:
The alignment of this icon could use a little tweaking - looks too close to text text and a bit too low. Though we may not need to deal with the styling here. Our `Button` component knows how to render an `ExternalLink` styled like a button. It might even properly bind the `include-graphql-port` prop out of the box, would need to experiment a bit. 

![Screen Shot 2022-08-16 at 12 23 21 PM](https://user-images.githubusercontent.com/8340719/184930837-715e5ba2-47a7-4401-b221-8f943f839d35.png)